### PR TITLE
Remove vendors for adopters page

### PR DESCRIPTION
**Pull request by cartermp**
Follow-up -- removes the vendors

### CODE REVIEW

File: content/en/ecosystem/adopters.md [hunk 1]
```diff
@@ -1,39 +1,24 @@
 ---
 title: Adopters
-description: Organizations that have adopted OpenTelemetry
+description: End-user organizations that have adopted OpenTelemetry
```
Comment (chalin) [inline content/en/ecosystem/adopters.md:3]: How about:

```suggestion
description: Organizations that use OpenTelemetry
```

File: content/en/ecosystem/adopters.md [hunk 2]
```diff
@@ -1,39 +1,24 @@
...
 # prettier-ignore
 spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wandera Zocdoc
 # All spelling entries must be on a single line
 ---
 
-A non-exhaustive, alphabetized list of organizations that have adopted
-OpenTelemetry.
+A non-exhaustive, alphabetized list of [end user organizations](https://community.cncf.io/end-user-community/) that have adopted OpenTelemetry. Note that to be an end user for OpenTelemetry, an organization cannot sell an Observability product.
```
Comment (chalin) [inline content/en/ecosystem/adopters.md]:
```suggestion
A non-exhaustive, alphabetized list of [end-user organizations](https://community.cncf.io/end-user-community/) that have adopted OpenTelemetry. Also see [Vendors](../vendors/).
```

File: content/en/ecosystem/adopters.md [hunk 3]
```diff
@@ -1,39 +1,27 @@
...
+description: Organizations that use OpenTelemetry
 # prettier-ignore
 spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wandera Zocdoc
 # All spelling entries must be on a single line
 ---
 
-A non-exhaustive, alphabetized list of organizations that have adopted
-OpenTelemetry.
+A non-exhaustive, alphabetized list of
+[end user organizations](https://community.cncf.io/end-user-community/) that
```
Comment (chalin) [inline content/en/ecosystem/adopters.md:10]:
```suggestion
[end-user organizations](https://community.cncf.io/end-user-community/) that
```

File: content/en/ecosystem/adopters.md [hunk 4]
```diff
@@ -1,39 +1,27 @@
...
+have adopted OpenTelemetry. Note that to be an end user for OpenTelemetry, an
+organization cannot sell an Observability product.
```
Comment (chalin) [inline content/en/ecosystem/adopters.md:12]:
```suggestion
have adopted OpenTelemetry. Also see [Vendors](../vendors/).
```

File: content/en/ecosystem/adopters.md (hunk 2 above)
Comment (chalin) [reply content/en/ecosystem/adopters.md]: I've reapplied these suggestions to the updated file, and so marking as resolved.
PTAL

**Review by svrnm — APPROVED** [review]

**Review by chalin — APPROVED** [review]
LGTM.

One question though: why not hyphenate "end-user" in "end user organizations"?

**Comment by svrnm** [comment]
> LGTM.
> 
> One question though: why not hyphenate "end-user" in "end user organizations"?

isn't that how the CNCF is doing it, e.g. https://community.cncf.io/:

> NCF brings together the world’s top developers, **end users**, and ...

**Comment by chalin** [comment]
@svrnm - ah, that's different because "end user" in that case isn't qualifying anything else. In English we hyphenate (when appropriate) related adjectives in compound adjectives. But, you don't have to take my word for it, ask your friendly neighbourhood AI ;) ...

> The usage of a hyphen in "end-user communities" follows the convention of using hyphens in compound adjectives. 
> 
> Compound adjectives are two or more words that together describe a noun. When the words are linked together by hyphens, they signal to the reader that they function as a single descriptor. Without the hyphen, the meaning might be unclear. In this case, "end-user" is functioning as an adjective describing "communities". Without the hyphen, it could be misread as communities that are at the "end" and also "user communities", which is different from the intended meaning. 
> 
> So, the rule that explains why we hyphenate "end user" in "end-user communities" is the rule of using hyphens in compound adjectives.

**Comment by svrnm** [comment]
@chalin, thanks for clarification!

**Comment by cartermp** [comment]
I actually originally wrote "end-user" (I too hyphenate), but removed it because CNCF didn't. I added it back in, though.

**Review by chalin — APPROVED** [review]
✨

**Pull request merged**